src/components/FooterModal.tsx

```diff
@@ -0,0 +1,28 @@
+import { Modal, ModalBody, ModalContent, ModalHeader } from '@nextui-org/react';
+
+interface FooterModalProps {
+	title: string;
+	content: string;
+	isOpen: boolean;
+	onClose: () => void;
+}
+
+const FooterModal: React.FC<FooterModalProps> = ({
```

Comment:
use `const Component = (props: Props) => {}` instead of `const Comoponent: React.FC<Props>`. https://kentcdodds.com/blog/how-to-write-a-react-component-in-typescript